Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py

```diff
@@ -1756,7 +1756,7 @@ def fetch_incidents():
     elif demisto.command() == 'recorded-future-get-alerts':
         get_alerts_command()
 
-except Exception, e:
+except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.